io/post_fv3.F90

```diff
@@ -3384,6 +4625,51 @@ subroutine set_postvars_fv3(wrt_int_state,grid_id,mype,mpicomp)
         end do
       end do
 
+      if(modelname=='FV3R') then
+        ! smoke and dust extinction
+        !$omp parallel do default(none) private(i,j,l) shared(lm,jsta,jend,ista,iend,zint,taod5503d,aextc55,extsmoke,extdust,spval)
+          do l=1,lm
+            do j=jsta,jend
+              do i=ista, iend
+                if(taod5503d(i,j,l)<spval.and.aextc55(i,j,l)<spval) then
+                  taod5503d(i,j,l)=extsmoke(i,j,l)+extdust(i,j,l)
+                  aextc55(i,j,l)=taod5503d(i,j,l)/(zint(i,j,l)-zint(i,j,l+1))
+                endif
+              enddo
+            enddo
+          enddo
+
+             !$omp parallel do default(none) private(i,j) shared(jsta,jend,ista,iend,spval,accswe_ice,accswe_land,acsnow)
+             do j=jsta,jend
+               do i=ista, iend
+                 if(accswe_ice(i,j)<spval .and. accswe_land(i,j)<spval) then
+                   acsnow(i,j) = accswe_ice(i,j) + accswe_land(i,j)
+                 else
+                   acsnow(i,j) = spval
+                 endif
+               enddo
+             enddo
+
+            !$omp parallel do default(none) private(i,j) shared(jsta,jend,ista,iend,spval,snacc_ice,snacc_land,sndepac)
+            do j=jsta,jend
+              do i=ista, iend
+                if(snacc_ice(i,j)<spval .and. snacc_land(i,j)<spval) then
+                  sndepac(i,j) = snacc_ice(i,j) + snacc_land(i,j)
+                else
+                  sndepac(i,j) = spval
+                endif
+              enddo
+            enddo
+
+      deallocate(extsmoke)
+      deallocate(extdust)
+      deallocate(buf)
+      deallocate(buf2)
+      deallocate(buf3)
+      deallocate(buf4)
```

Comment:
There are no buf, buf2, buf3 and buf4 anymore.

Reply:
@DusanJovic-NOAA Thanks for catching that.